ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDialogOptions.kt

```diff
@@ -86,8 +86,8 @@ data class PaywallDialogOptions internal constructor(
             this.listener = listener
         }
 
-        fun setMyAppPurchaseLogic(myAppPurchaseLogic: MyAppPurchaseLogic?) = apply {
-            this.myAppPurchaseLogic = myAppPurchaseLogic
+        fun AppPurchaseLogic(purchaseLogic: PurchaseLogic?) = apply {
```

Comment:
I think this is supposed to be
```suggestion
        fun setPurchaseLogic(purchaseLogic: PurchaseLogic?) = apply {
```

Reply:
@tonidero This would then have the same comment as https://github.com/RevenueCat/purchases-android/pull/1814/files#r1724558113 I assume?

Reply:
Yes that’s correct! That comment applies here too